simple_scripts/readme.md

```diff
@@ -0,0 +1,88 @@
+# Faraday - Simple Scripts
```

Comment:
Please note that these are largely development scripts and not meant for mainstream use at this time. Basically they are only available if you checkout git/download them manually and we assume you know what you're doing if you do that.